STNetwork/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/network/requests/TransferRequest.kt

```diff
@@ -51,6 +52,16 @@ internal class TransferRequest(
         )
     }
 
+    suspend fun deleteTransfer(jsonBody: JsonObject): Boolean {
```

Comment:
I think we'd rather have the `linkUuuid` and `token` parameters specified explicitly here, or a typed object containing them, in place of an unspecific `JsonObject` that doesn't need to be dynamic.